synadm/cli/user.py

```diff
@@ -374,6 +382,10 @@ def modify(ctx, helper, user_id, password, password_prompt, display_name,
             "Deactivating a user and setting a password doesn't make sense.")
         raise SystemExit(1)
 
+    if clear_user_type and user_type:
+        click.echo("Use either --user-type or --clear-user-type, not both.")
+        raise SystemExit(1)
+
```

Comment:
Instead of this sanity check, please use the mutual exclusive option group. Look at the imports of cli/modules to find its exact name.

Reply:
If i add them to a `MutuallyExclusiveOptionGroup` and i only pass the `--user-type` option to the `user modify` command, it tells me that i have to pass one of the other options (as if the mutually exclusive group is not part of the `RequireAnyOptionGroup`).

Reply:
Ah… from the [documentation](https://click-option-group.readthedocs.io/en/latest/tutorial.html#limitations) of click-option-group:

> The package does not support nested option groups (option subgroups). This is intentional. Nested option groups complicate the implementation, API and CLI and most often it is not necessary.

So it seems either the current implementation remains, or the user type change can only be done together with changing anything else.

Reply:
Ah ok then this is the reason why the password thing right above is done with a conditian as well. Ok then please leave as as. Not worth the hassle! :-)